docs/CHANGELOG.rst

```diff
@@ -6,6 +6,21 @@ All notable changes to this project are documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 
 
+==========
```

Comment:
Since we don't version this repo the same as resolwe-bio is, there is no need for unreleased section. Simply add the date tag and list changes below. Look at some previous commits for examples.